draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -1034,6 +1054,23 @@ If a subflow attempts to join an existing MP-DCCP connection, but MP-DCCP option
 are not present in the handshake messages or the protocol version doesn't
 match the value negotiated for the first flow, that subflow will be closed.
 
+## Congestion Control Considerations
+
+Senders MUST manage per-path congestion status, and SHOULD avoid to send
+more data on a given path than congestion control on that path
+allows.
+
+When a Multipath DCCP connection uses two or more paths, there is no
+guarantee that these paths are fully disjoint.  When two (or more
+paths) share the same bottleneck, using a standard congestion control
+scheme could result in an unfair distribution of the bandwidth with
+the multipath connection getting more bandwidth than competing single
+paths connections.  Multipath TCP uses the LIA congestion control
```

Comment:
... uses the coupled congestion control scheme 'Linked Increases Algorithm' specified in ...

Reply:
https://github.com/markusa/ietf-multipath-dccp/pull/54/commits/2f016c6b698dd809ea2d8d81a5b3129b3eb6cb29